packages/framework/src/Model/Order/Item/OrderItemPriceCalculation.php

```diff
@@ -31,15 +39,34 @@ public function __construct(
      * @param int $domainId
      * @return \Shopsys\FrameworkBundle\Component\Money\Money
      */
-    public function calculatePriceWithoutVat(OrderItemData $orderItemData, int $domainId): Money
```

Comment:
maybe this method could also remain the same, and the decision could be made inside (the two new methods could be protected then), or? It would simplify the usage imho

Reply:
This class is trying to be as easiest for slight change in calculations, that I wanted to keep that - I started with merging all those small methods into one as I didnt see the need for that, but then I realized that and keep the code splitted in as many small parts as possible.